packages/clerk-js/src/ui/components/SignIn/SignInFactorOneCodeForm.tsx

```diff
@@ -36,6 +38,8 @@ export const SignInFactorOneCodeForm = (props: SignInFactorOneCodeFormProps) =>
   const { setActive } = useClerk();
   const supportEmail = useSupportEmail();
   const clerk = useClerk();
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
❓ Is there any reason we can't use `useCardState` for loading state here?

Reply:
Oops! I missed that, I will refactor it.